src/core_modules/capture-core/components/Pages/MainPage/TeiWorkingLists/types/concept.types.js

```diff
@@ -19,3 +21,25 @@ export type TeiWorkingListsTemplate = {
 };
 
 export type TeiWorkingListsTemplates = Array<TeiWorkingListsTemplate>;
+
+export type ColumnConfigBase = {|
+    id: string,
+    visible: boolean,
+    type: $Values<dataElementTypes>,
+    header: string,
+    options?: ?Array<{text: string, value: any}>,
+    multiValueFilter?: boolean,
+|};
+export type MetadataColumnConfig = {
+    ...ColumnConfigBase,
+};
```

Comment:
I understand that the way you go about it is that you have this type open for future extension. Is it a problem to annotate exactly here?
```suggestion
export type MetadataColumnConfig = {
    ...ColumnConfigBase,
};
```

Reply:
This will be passed on to the `WorkingLists` and flow will complaint if I annotate this one exact (feel free to try :))